find-files-to-check.sh

```diff
@@ -3,4 +3,5 @@
 find . -type f                  					\
     -not -path "./project_liberation/settings/*" 	\
     -not -path "*/migrations/*"             		\
+    -not -path "./project_liberation/manage.py"     \
```

Comment:
This is not a path, this is just single file. I'm gonna fix it on my own, but please remember that it should look like this:

```bash
find . -type f                  			\
    -not -path "./project_liberation/settings/*" 	\
    -not -path "*/migrations/*"             		\
    -not -name "manage.py"     				\
    -name "*.py"                            		\
```